docs/en/enterprise-edition/rn/known-issues/known-fixed-issues.adoc

```diff
@@ -19,6 +19,13 @@ The list of fixed issues are not cumulative; only the issues that are fixed with
 |*ISSUE ID*
 |*DESCRIPTION*
 
+|tt:[Fixed in 31.02.137]
```

Comment:
@Pubs-MV this has to be thought through. This is a placeholder to see how this maintenance blurb fits in here.

Reply:
let' remove this